patch/attiny841.yaml

```diff
@@ -0,0 +1,5 @@
+_svd: ../svd/attiny841.svd
+
+_include:
+  - "common/port.yaml"
```

Comment:
Are there really no other patches from `common/` which apply for this MCU?

Reply:
I'll look into this some more